testfiles/EMSThermochromicWindow.idf

```diff
@@ -6014,7 +6014,7 @@
 
   EnergyManagementSystem:ProgramCallingManager,
     My thermochromic window emulator,  !- Name
-    BeginTimestepBeforePredictor,  !- EnergyPlus Model Calling Point
+    BeginZoneTimestepBeforeInitHeatBalance,  !- EnergyPlus Model Calling Point
```

Comment:
Will you expect any diffs because of this?

Reply:
Yes, but now I wonder if I should leave that for a different PR and just add the calling point here.

Reply:
Nah, CI has already started working on it.  Let it finish as-is so it can merge in.  I'll anticipate diffs in these two files and if we need to investigate further, we'll do it after we freeze the IDD.